packages/client/bin/cli.ts

```diff
@@ -382,6 +387,21 @@ const args: ClientOpts = yargs(hideBin(process.argv))
     boolean: true,
     default: true,
   })
+  .option('statelessVerkle', {
+    describe: 'Run verkle+ hardforks using stateless verkle stateManager',
+    boolean: true,
+    default: true,
+  })
```

Comment:
Please add some experimental notes at least for the two verkle flags.

(are the two others meant to make it into the main code base?)

Reply:
yes, we can include other two as well as they are handy for debugging for merkle/stateful verkle (whenever that happens) and don't really have anything specific to stateless verkle